guppylang/prelude/builtins.py

```diff
@@ -57,11 +59,12 @@ class nat:
     """Class to import in order to use nats."""
 
 
-class array:
-    """Class to import in order to use arrays."""
+_S = TypeVar("_S")
+_T = TypeVar("_T")
+
 
-    def __class_getitem__(cls, item):
-        return cls
+class array(Generic[_S, _T]):
+    """Class to import in order to use arrays."""
```

Comment:
The names are switched from the `Array` definition below (which uses `arrar[T, n]`).
Can you document which typevar is the array type and which one the length?

Reply:
This class is just for the purpose of mypy and ruff to stop them from complaining, so it really doesn't matter what type vars we use here. But you're right, I'll choose some more suggestive names 👍